modules/nw-multus-bridge-object.adoc

```diff
@@ -17,9 +18,11 @@ plugin:
 |`string`
 |The CNI specification version. The `0.3.1` value is required.
```

Comment:
0.3.1 is "required" or the minimum? In the example below "0.4.0" is used.

Reply:
This is from the OCP docs directly. @pmtk Do you know the answer to this one?

Reply:
I'd use 0.4.0

Reply:
I see, our example object config doesn't match the original table. I updated the table row for MicroShift to be consistent with our example, which is `0.4.0`. I suspect the OCP doc simply hasn't been updated in a long time. @s1061123 I can move the OCP examples up to `0.4.0` if you think warranted; otherwise I will leave those.